tools/ci/complexity.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+function usage {
+    cat <<USAGE
+Usage: $0 [files...]
+
+A script to run complexity in CI. This is needed to avoid issues with
+exit codes.
+
+USAGE
+}
+
+if [ $# -eq 0 ] ; then
+    usage
+    exit 1
+fi
+
```

Comment:
You could also consider checking to make sure that `complexity` is installed first, e.g.:

```sh
which complexity>/dev/null
if [ $? != 0 ]
then
    echo "complexity is not installed"
    exit 1
fi
```

Reply:
done